corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -309,6 +309,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         ui: {
             actionButton: '.caselist-action-button button',
             searchButton: '#case-list-search-button',
+            searchTextBox: '.input-group',
```

Comment:
`.input-group` is a pretty generic selector that could be used elsewhere on the page in the future. Could this selector be `.module-search-container` (or `.module-search-container .input-group` if needed)?

Reply:
Yes, `.module-search-container` works, so I changed it to that.